lmsadmin.sh

```diff
@@ -0,0 +1,23 @@
+# Janet Fernando
+# Admin User Data for AWS and LB 
+
+#!/bin/bash
+
+sudo yum install git -y
+sudo yum install java-1.8.0-openjdk -y
+sudo yum install maven -y

```

Comment:
A simple script to store these first few commands would be nice in the future to be used for other EC2 instance User data